pkg/ssh/ssh.go

```diff
@@ -494,6 +494,34 @@ func RunSSHUploadStakingFiles(host *models.Host, nodeInstanceDirPath string) err
 	)
 }
 
+// RunSSHRenderAvagoAliasConfigFile renders avalanche alias config to a remote host via SSH.
+func RunSSHRenderAvagoAliasConfigFile(
```

Comment:
pls make this to work with already created aliases. I mean, the file may already exists and contain aliases
from a previous setup for a different blockchain

Reply:
k will do